tools/src/main/scala/scala/scalanative/codegen/CodeGen.scala

```diff
@@ -101,6 +105,7 @@ object CodeGen {
       }
 
       dir.merge(Seq(body), headers)
+      headers
```

Comment:
maybe make it clear and split herer to `-body.ll` and `-herader.ll` and merge them into `id.ll`?

Reply:
I really didn't want to mess with that because we really just need the path to avoid all the look ups.

Reply:
ok, it was just an idea.

Reply:
I think right now we are just improving and hardening the code so we are in good territory.

Reply:
@catap If you see improvements in the code there maybe you should add an issue.